src/main/java/uk/ac/ebi/eva/pipeline/listeners/VariantOptionsConfigurerListener.java

```diff
@@ -33,27 +33,24 @@
 
     private static final Logger logger = LoggerFactory.getLogger(VariantOptionsConfigurerListener.class);
     private final boolean includeSamples;
-    private final boolean compressGenotypes;
     private final boolean calculateStats;
     private final boolean includeStats;
 
     @Autowired
     private JobOptions jobOptions;
 
     public VariantOptionsConfigurerListener(boolean includeSamples,
-                                            boolean compressGenotypes,
                                             boolean calculateStats,
                                             boolean includeStats) {
         this.includeSamples = includeSamples;
-        this.compressGenotypes = compressGenotypes;
         this.calculateStats = calculateStats;
         this.includeStats = includeStats;
     }
 
     @Override
     public void beforeJob(JobExecution jobExecution) {
         logger.debug("Setting up job " + jobExecution.getJobInstance().getJobName());
-        jobOptions.configureGenotypesStorage(includeSamples, compressGenotypes);
+        jobOptions.configureGenotypesStorage(includeSamples);
         jobOptions.configureStatisticsStorage(calculateStats, includeStats);
```

Comment:
Same as with `includeSamples` and `compressGenotypes`, I think the pair `calculateStats` and `includeStats` always have the same value: if statistics are calculated then they are "included" in the database and vice-versa. But maybe we don't want to get into such a refactoring for this PR.